src/registry.jl

```diff
@@ -489,3 +489,5 @@ add_format(format"FCS", "FCS", [".fcs"], [:FCSFiles => UUID("d76558cf-badf-52d4-
 add_format(format"HTML", (), [".html", ".htm"], [MimeWriter, SAVE])
 
 add_format(format"MIDI", "MThd", [".mid", ".midi", ".MID"], [:MIDI => UUID("f57c4921-e30c-5f49-b073-3f2f2ada663e")])
+
+add_format(format"QOI", "qoif", ".qoi", [:QOI => UUID("4b34888f-f399-49d4-9bb3-47ed5cae4e65")])
```

Comment:
```suggestion
add_format(format"QOI", "qoif", ".qoi", [:QOI => UUID("4b34888f-f399-49d4-9bb3-47ed5cae4e65")], [idImageIO])
```
https://github.com/JuliaRegistries/General/pull/51497

I'll let you decide whether you want to register QOI.jl here and then we can merge.